CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.1...HEAD)
 
 ### Backward Compatibility Fixes
+- Use short array syntax `[]` instead of long `array()` - so break compatibility with PHP 5.3
```

Comment:
@merk, You show me @ruflin's commit at 12 November 2015, but 10 May 2016 @ruflin write in my PR - add `break with PHP 5.3`.
I think need wait for @ruflin